vm-mock/vm.js

```diff
@@ -174,14 +177,15 @@ export default function createMockedABI(memory, createImports, instantiate, bina
       },
 
       assembly_script_get_call_stack() {
-        return webModule.__newArrayBuffer('[ ' + callStack + ']');
+        return '[ ' + callStack + ']';
```

Comment:
still not working on my side =>     generateEvent(`${Context.caller().toByteString()}`); return null